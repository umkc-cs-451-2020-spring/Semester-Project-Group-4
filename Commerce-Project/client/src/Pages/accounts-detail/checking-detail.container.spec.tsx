import React from 'react';
import { shallow } from 'enzyme';
import CheckingDetail from './checking-detail.container';

describe('Checking Detail', () => {

  it('Render the Checking Detail', () => {
    const component = shallow(<CheckingDetail />);

    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
