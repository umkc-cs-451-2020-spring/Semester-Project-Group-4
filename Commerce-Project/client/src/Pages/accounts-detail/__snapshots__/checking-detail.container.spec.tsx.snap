// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking Detail Render the Checking Detail 1`] = `
<WithStyles(ForwardRef(Paper))
  className="makeStyles-paper-1"
  elevation={0}
>
  <WithStyles(ForwardRef(Typography))
    variant="h5"
  >
    Checking Account
  </WithStyles(ForwardRef(Typography))>
  <WithStyles(ForwardRef(Divider))
    className="makeStyles-divider-2"
  />
  0
</WithStyles(ForwardRef(Paper))>
`;
